src/sgame/components/HealthComponent.cpp

```diff
@@ -238,7 +238,9 @@ Util::optional<Vec3> direction, int flags, meansOfDeath_t meansOfDeath) {
 
 	if (entity.oldEnt != source && source->client) {
 		bool lethal = (health <= 0);
-		CombatFeedback::HitNotify(source, entity.oldEnt, location, take, meansOfDeath, lethal);
+		vec3_t tmp;
+		location.value().Store( tmp );
```

Comment:
This will throw an exception if location is not present.

Hint hint, you could just leave this for direct replacement with GLM...